models/maespa/R/write.config.MAESPA.R

```diff
@@ -0,0 +1,83 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2012 University of Illinois, NCSA.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the 
+# University of Illinois/NCSA Open Source License
+# which accompanies this distribution, and is available at
+# http://opensource.ncsa.illinois.edu/license.html
+#-------------------------------------------------------------------------------
+
+##-------------------------------------------------------------------------------------------------#
+##' Writes a config file for Maespa
+##'
+##' Requires a pft xml object, a list of trait values for a single model run,
+##' and the name of the file to create
+##'
+##' @name write.config.MAESPA
+##' @title Write MAESPA configuration files
+##' @param settings list of settings from pecan settings file
+##' @param run.id id of run
+##' @return configuration file for MODEL for given run
+##' @export
+##' @author Tony Gardella
+##-------------------------------------------------------------------------------------------------#
+write.config.MAESPA <- function(settings,run.id){
```

Comment:
@tonygardella this line is wrong. All write.config functions have to have the same standard arguments in the same standard order. Just because you are not USING defaults and trait.values, doesn't mean they can stop being arguments.